tests/acceptance/premium/test_group_merge_request_approval_rules.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from tests.acceptance import run_gitlabform
+
+pytestmark = pytest.mark.requires_license
+
+
+class TestGroupMergeRequestApprovalRules:
+    def test__add_single_rule(self, group):
+
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_merge_requests_approval_rules:
+              any:
+                approvals_required: 5
+                rule_type: regular
+                name: "Any Approver"
+        """
+
+        run_gitlabform(config, group)
+        rules = group.approval_rules.list()
+        assert len(rules) == 1
+
+        for rule in rules:
+            assert rule.name == "Any Approver"
+            assert rule.approvals_required == 5
+            assert rule.rule_type == "regular"
```

Comment:
Just a minor comment or question. I'm not sure the for loop is giving any advantage. Even if we add more than 1 rules in the config, the assertions have hard-coded value. So, it won't be useful, right? Maybe let's just go with `rule[0].... = ...`?

Reply:
That's a good hint, I think the code there was actually meant for the multiple rules part. Removed the loop and added some additonal asserts.